app/components/shared/LoadingError.js

```diff
@@ -0,0 +1,32 @@
+import { FormattedMessage as T } from "react-intl";
```

Comment:
Can we change this file to be `.jsx` ?
I know all other components are with `.js` but we are trying to move away from this and keep components in `.jsx` files and pure JS in `.js` files.

Reply:
Of course, I really miss Typescript.